cuckoo/cuckoo_main.py

```diff
@@ -1116,6 +1116,12 @@ def _set_task_parameters(self, kwargs: Dict[str, Any], file_ext: str, parent_sec
         arguments = self.request.get_param("arguments")
         dump_memory = self.request.get_param("dump_memory")
         no_monitor = self.request.get_param("no_monitor")
+
+        # If the user didn't select no_monitor, but at the service level we don't want the monitor to run on
+        # Windows 10x64, then:
```

Comment:
I think this would read better as: "If the user didn't select no_monitor, but at the service level we want no_monitor on Windows 10x64, then:"